ompi/mpi/fortran/use-mpi-f08/get_elements.c.in

```diff
@@ -36,9 +36,11 @@ PROTOTYPE VOID get_elements(STATUS status, DATATYPE datatype, COUNT_OUT count)
 
         if (MPI_SUCCESS == c_ierr) {
             c_ierr = @INNER_CALL@(&c_status, c_type,
-                                      OMPI_SINGLE_NAME_CONVERT(count));
-            OMPI_SINGLE_INT_2_FINT(count);
+                                      &c_count);
         }
     }
+    if (MPI_SUCCESS == c_ierr) {
+        *count = (@COUNT_FINT_TYPE@)(c_count);
+    }
```

Comment:
Nit-pick: we're assigning to `*count` twice if `OMPI_IS_FORTRAN_STATUS_IGNORE(status)` is true (first time in line 32 above). Maybe this can be consolidated?

Reply:
nice catch.  refactored a bit.